Content.Server/_RMC14/Marines/MarineAnnounceCommand.cs

```diff
@@ -0,0 +1,34 @@
+using Content.Server.Administration;
+using Content.Shared.Administration;
+using Robust.Shared.Console;
+
+namespace Content.Server._RMC14.Marines;
+
+[AdminCommand(AdminFlags.Moderator)]
+public sealed class MarineAnnounceCommand : IConsoleCommand
+{
+    public string Command => "marineannounce";
+    public string Description => Loc.GetString("rmc-command-marineannounce-description");
+    public string Help => Loc.GetString("rmc-command-marineannounce-help");
+
+    public void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        var _marineAnnounce = IoCManager.Resolve<IEntitySystemManager>().GetEntitySystem<MarineAnnounceSystem>();
+        if (args.Length == 0)
+        {
+            shell.WriteError("Not enough arguments! Need at least 1.");
+            return;
+        }
+
+        if (args.Length == 1)
+        {
+            _marineAnnounce.AnnounceHighCommand(args[0]);
+        }
+        else
+        {
+            var message = string.Join(' ', new ArraySegment<string>(args, 1, args.Length-1));
```

Comment:
This is surprisingly the first usage of ArraySegment in content but I guess it's not wrong, you should be able to do this instead though but test it in-game if you can
```suggestion
            var message = string.Join(' ', args[1..]);
```

Reply:
works.

![зображення](https://github.com/user-attachments/assets/6cdfc306-3f9b-43eb-b491-4603d23def41)